icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -623,6 +623,8 @@ class Storage:
         bucket: str,
         prefix: str | None,
         credentials: AnyS3Credential | None = None,
+        *,
+        use_object_store: bool = False,
```

Comment:
why don't we create a new function that we don't document to create this?

Reply:
Cool I'll do that instead